RELEASE_INSTRUCTIONS.md

```diff
@@ -0,0 +1,33 @@
+# Release Instructions
+
+## TL;DR
+
+1. Trigger the `Version` workflow specifying the `version-bump`
```

Comment:
Perhaps we can add the link to the workflow: https://github.com/onebeyond/cuckoojs/actions/workflows/version.yml